exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+In this exercise you will be building error handling for a simple integer calculator. 
+To make matters simple, methods for calculating addition, multiplication and division are provided.
+
+The goal is to have a working calculator that returns a String with the following pattern: `16 + 51 = 67`, when provided with arguments `16`, `51` and `+`.
+
+
+```java
+CalculatorConundrum obj = new CalculatorConundrum();
+
+obj.calculate(16, 51, "+"); // => returns "16 + 51 = 67"
+
+obj.calculate(32, 6, "*"); // => returns "32 * 6 = 192"
+
+obj.calculate(512, 4, "/"); // => returns "512 / 4 = 128"
+```
+
+## 1. Implement the method calculate to support a few basic operations 
+
+The main method for implementation in this task will be the `CalculatorConundrum.calculate()` method. 
+It takes three arguments. 
+The first two arguments are integer numbers on which an operation is going to be conducted. 
+The third argument is of type String and for this exercise it is necessary to implement the following operations:
+
+- addition using the `+` String
+- multiplication using the `*` String
+- division using the `/` String
+
+## 2. Handle illegal operations
+
+All the following cases need to throw an `IllegalOperationException`:
+
+* when the `operation` argument is `null`
+* when the `operation` is the empty String
+* when the `operation` argument is anything other than `+`, `*`, or `/`
```

Comment:
Since students may not have access to the tests (eg. when using the website editor), we should explicitly say what error message to use in the instructions here.